ajax/libs/twbs-pagination/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "twbs-pagination",
+  "filename": "jquery.twbsPagination.min.js",
+  "version": "1.3.1",
+  "homepage": "https://github.com/esimakin/twbs-pagination",
+  "authors": [
+    "Eugene Simakin <eugenesimakin@mail.ru>"
+  ],
+  "description": "jQuery simple pagination plugin for bootstrap-style webpages",
+  "dependencies": {
+    "jquery": ">=1.7.0"
+  },
+  "keywords": [
+    "pagination",
+    "jQuery",
+    "jQuery-plugin",
+    "bootstrap"
+  ],
+  "license": "Apache 2.0",
```

Comment:
Hi, @PeterDaveHello ! Does this related to https://github.com/esimakin/twbs-pagination ? We can fix this, but what exactly is wrong?